{"uuid": "70832638-c2fe-4dbe-aabf-0e4a837d707c", "children": ["7b6152a2-168a-4365-95ac-3b49edebc076"], "befores": [{"name": "user_recharge", "status": "broken", "statusDetails": {"message": "TypeError: 'function' object is not subscriptable\n", "trace": "  File \"F:\\python3.7\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"F:\\python3.7\\lib\\site-packages\\_pytest\\fixtures.py\", line 957, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"F:\\python3.7\\lib\\site-packages\\_pytest\\fixtures.py\", line 783, in call_fixture_func\n    res = next(it)\n  File \"F:\\ningmengban\\NMB1\\web3\\Test_case\\test_4_recharge.py\", line 18, in user_recharge\n    assert user_recharge[2].current_url == Recharge_datas.recharge_url\n"}, "start": 1578474664471, "stop": 1578474665972}], "start": 1578474664471, "stop": 1578474666142}